docs/upgrades.md

```diff
@@ -10,7 +10,7 @@ Sematic has 3 versioned entities, depending on your usage:
 - **pip package**: If you are using Sematic only locally, this is
 the only versioned entity you need. This determines the code
 that will be used for your code to interact with the server,
-including how Sematic resolvers interact with the server.
+including how Sematic runners interact with the server.
```

Comment:
Please add a `v0.31.2 to v0.32.0` section to this file to officially inform users of the change, and to explain that they have to perform the rename / import change in their code, and to switch from `future.resolve(resolver)` to `runner.run(future)`, as it will be deprecated in the future.

Please also make a changelog entry at the same time, pointing to this upgrade entry.

Reply:
Did this [here](https://github.com/sematic-ai/sematic/pull/981/files#diff-2b35cda5da63a0dd7e1834e0063af64826711909facdb8a56fafece345fa24fe). Comment no longer required.